iModelCore/iModelPlatform/DgnCore/SimplifyGraphic.cpp

```diff
@@ -860,6 +861,7 @@ void SimplifyGraphic::_AddBody(IBRepEntityCR geom)
 +---------------+---------------+---------------+---------------+---------------+------*/
 void SimplifyGraphic::_AddTextString(TextStringCR text)
     {
+//*q*/printf("SimplifyGraphic::_AddTextString\n");

```

Comment:
Remove?

Reply:
oops, thought I had gotten all of those.